Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -338,15 +367,87 @@ public function renderThumbnailsCommand(int $limit = null, bool $quiet = false)
                 break;
             }
         }
-        !$quiet && $this->output->progressFinish();
+    }
+
+    /**
+     * Render asset variants
+     *
+     * Loops over missing configured asset variants and renders them. Optional ``limit`` parameter to
+     * limit the amount of variants to be rendered to avoid memory exhaustion.
+     *
+     * If the re-render parameter is given, any existing variants will be rendered again, too.
+     *
+     * @param integer $limit Limit the amount of variants to be rendered to avoid memory exhaustion
+     * @param bool $quiet If set, only errors will be displayed.
+     * @param bool $recreate
+     * @return void
+     * @throws AssetVariantGeneratorException
+     * @throws IllegalObjectTypeException
+     */
+    public function renderVariantsCommand($limit = null, bool $quiet = false, bool $recreate = false): void
+    {
+        $resultMessage = null;
+        $generatedVariants = 0;
+        $configuredVariantsCount = 0;
+        $configuredPresets = $this->assetVariantGenerator->getVariantPresets();
+        foreach ($configuredPresets as $configuredPreset) {
+            $configuredVariantsCount += count($configuredPreset->variants());
+        }
```

Comment:
nitpick:
We _could_ add a check here for "no asset presets are defined" (just cosmetics of course):
```suggestion
        }
        if ($configuredVariantsCount === 0) {
            $this->outputLine('There are no image variant presets configured, exiting…');
            $this->quit();
        }
```